src/oas2/transformers/securities.ts

```diff
@@ -76,6 +76,8 @@ export function translateToSingleSecurity(security: DeepPartial<Security>) {
   return;
 }
 
-export function translateToSecurities(securities: Array<DeepPartial<Security>>): HttpSecurityScheme[] {
-  return compact(securities.map(translateToSingleSecurity));
+export function translateToSecurities(securities: Security[][]): HttpSecurityScheme[][] {
+  return securities.map(security => {
```

Comment:
You can make this one line

`return securities.map(security => compact(security.map(translateToSingleSecurity)))`